src/factories/OrchestratorFactory.sol

```diff
@@ -61,7 +64,11 @@ contract OrchestratorFactory is IOrchestratorFactory {
     //--------------------------------------------------------------------------
     // Constructor
 
-    constructor(address target_, address moduleFactory_) {
+    constructor(
+        address target_,
+        address moduleFactory_,
+        address _trustedForwarder //@todo should this be changeable?
```

Comment:
Yeah I wanted to ask that the reviewees ^^
"as long as it works" is the key argument here. Because the Forwarder basically circumvents the Authorization, by customizing the msg.sender we have to be careful here. I think this directly ties into the topic of safe module upgrades.
Also: Who does the updating of the forwarder address? If its each own orchestrator this has to be done by each owner seperately. A beacon system might make sense here.